### ISSUE TITLE
ScoredPoint cannot be serialized by `model_dump()` method due to float32 type contained in result

### ISSUE THREAD

**Issue by chan150** [qdrant/qdrant-client#334]
Some query result of `Qdrant-client` return List of ScoredPoint type.
However, it cannot be serialized to json (Dict) since they have float32 type.

They should be converted float inside `model_dump()` method called.

Here, I have attached a test code used:
```python
result = self.client.recommend(
  collection_name=collection_name,
  positive=ids,
  strategy=models.RecommendStrategy.AVERAGE_VECTOR,
  # query_vector=concat,
  query_filter=models.Filter(
      must=[
          models.FieldCondition(
              key='label',
              match=models.MatchValue(
                  value=label
              ),
          ),
      ],
      must_not=[
          models.HasIdCondition(has_id=ignores),
      ],
  ),
  with_payload=False,
  with_vectors=False,
  score_threshold=score_threshold
)
return [e.model_dump() for e in result]
```

However, it is failed with the error message:
`TypeError: Object of type float32 is not JSON serializable`

It is solved with a quick fix but ugly:
```python
return [json.loads(e.model_dump_json()) for e in result]
```

Python client version: 1.6.0

**Comment by joein** [comment]
Hi @chan150 

The issue is related to local mode, right? 
(Local mode is used when you initialize your client with `QdrantClient(":memory:")` or `QdrantClient(path='...')`)

I've made a fix for local mode #337

**Comment by chan150** [comment]
Hi @joein 
It is solved with a fix #337 

Thank you for the fix.

**Comment by joein** [comment]
the fix available as of 1.6.1
